PGM/src/main/java/tc/oc/pgm/polls/types/PollKick.java

```diff
@@ -0,0 +1,44 @@
+package tc.oc.pgm.polls.types;
+
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.assistedinject.AssistedInject;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.entity.Player;
+import tc.oc.commons.bukkit.chat.PlayerComponent;
+import tc.oc.commons.bukkit.nick.IdentityProvider;
+import tc.oc.commons.core.chat.Audiences;
+import tc.oc.commons.core.chat.Component;
+import tc.oc.pgm.polls.Poll;
+import tc.oc.pgm.polls.PollManager;
+
+public class PollKick extends Poll {
+
+    public interface Factory {
+        PollKick create(String initiator, Player player);
+    }
+
+    private final Player player;
+    private final IdentityProvider identityProvider;
+
+    @AssistedInject PollKick(@Assisted String initiator, @Assisted Player player, PollManager pollManager, Audiences audiences, IdentityProvider identityProvider) {
+        super(pollManager, initiator, audiences);
+        this.player = player;
+        this.identityProvider = identityProvider;
+    }
+
+    @Override
+    public void executeAction() {
+        if(player != null) player.kickPlayer(ChatColor.DARK_RED + "You were poll-kicked.");
```

Comment:
There needs to be a better explanation for the kick. Also, I would implement this via an off-record punishment so they get all the formatting. 

Although, you may need to make sure off-record punishments, instead of saying ...appeal here... they say ...punishment does not count... or something.